create_gz_vendor_pkg/templates/vendor.sh.in

```diff
@@ -0,0 +1,3 @@
+if [ -d "$AMENT_CURRENT_PREFIX/opt/@PROJECT_NAME@/share/gz" ]; then
+  ament_prepend_unique_value GZ_CONFIG_PATH "$AMENT_CURRENT_PREFIX/opt/@PROJECT_NAME@/share/gz"
```

Comment:
I tested this by building a colcon workspace with the vendor packages and deleting the `.dsv` files in the `install` directories. What I found is that since `$AMENT_CURRENT_PREFIX` was `/opt/ros/rolling`, the condition here is never satisfied and `GZ_CONFIG_PATH` is never set. So I guess this would only work if you're installing the vendor packages to `/opt/ros/<ros-version>`?